tests/base.py

```diff
@@ -65,16 +65,8 @@ def get_credentials():
             'client_secret': os.getenv('TAP_SQUARE_APPLICATION_SECRET'),
         }
 
-    @staticmethod
-    def expected_check_streams():
-        return {
-            'items',
-            'categories',
-            'discounts',
-            'taxes',
-            'employees',
-            'locations',
-        }
+    def expected_check_streams(self):
+        return set(self.expected_metadata().keys()).difference(set())
```

Comment:
This is a duplicate of #18

Reply:
yeah #18 wasn't ready yet so i threw it in here too